[2025-02-11 11:00:52,374] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: grippe_ETL_pipeline.load_vaccins2024 scheduled__2025-02-11T10:00:00+00:00 [queued]>
[2025-02-11 11:00:52,390] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: grippe_ETL_pipeline.load_vaccins2024 scheduled__2025-02-11T10:00:00+00:00 [queued]>
[2025-02-11 11:00:52,390] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2025-02-11 11:00:52,391] {taskinstance.py:1357} INFO - Starting attempt 1 of 4
[2025-02-11 11:00:52,391] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2025-02-11 11:00:52,414] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): load_vaccins2024> on 2025-02-11 10:00:00+00:00
[2025-02-11 11:00:52,420] {standard_task_runner.py:52} INFO - Started process 314 to run task
[2025-02-11 11:00:52,426] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'grippe_ETL_pipeline', 'load_vaccins2024', 'scheduled__2025-02-11T10:00:00+00:00', '--job-id', '44', '--raw', '--subdir', 'DAGS_FOLDER/ETL_Vaccin.py', '--cfg-path', '/tmp/tmpja3_usw8', '--error-file', '/tmp/tmpjkjkh26q']
[2025-02-11 11:00:52,427] {standard_task_runner.py:80} INFO - Job 44: Subtask load_vaccins2024
[2025-02-11 11:00:52,518] {task_command.py:370} INFO - Running <TaskInstance: grippe_ETL_pipeline.load_vaccins2024 scheduled__2025-02-11T10:00:00+00:00 [running]> on host 976dfa0d1440
[2025-02-11 11:00:52,628] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=grippe_ETL_pipeline
AIRFLOW_CTX_TASK_ID=load_vaccins2024
AIRFLOW_CTX_EXECUTION_DATE=2025-02-11T10:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2025-02-11T10:00:00+00:00
[2025-02-11 11:00:53,524] {logging_mixin.py:115} INFO - ✅ Chargement réussi dans la table grippe_vaccins2024 (52 lignes)
[2025-02-11 11:00:53,525] {python.py:173} INFO - Done. Returned value was: None
[2025-02-11 11:00:53,549] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=grippe_ETL_pipeline, task_id=load_vaccins2024, execution_date=20250211T100000, start_date=20250211T110052, end_date=20250211T110053
[2025-02-11 11:00:53,592] {local_task_job.py:156} INFO - Task exited with return code 0
[2025-02-11 11:00:53,644] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
